localization-proofreading-english/StringTableEnSymptoms.xml

```diff
@@ -96,17 +96,17 @@
             <GameDBLocalizedString>      <LocID>SYM_DIARRHEA</LocID>                                        <Text>Diarrhea</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>SYM_DIARRHEA_DESCRIPTION</LocID>                            <Text>Frequent bowel movements, liquid stool.</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>SYM_DIARRHEA_DESCRIPTION</LocID>                            <Text>The patient has frequent bowel discharges with faeces in liquid form.</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>SYM_DIMINISHED_HEARING</LocID>                              <Text>Diminished hearing</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>SYM_DIMINISHED_HEARING_DESCRIPTION</LocID>                  <Text>Decreased level of hearing.</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>SYM_DIMINISHED_HEARING_DESCRIPTION</LocID>                  <Text>The patient does not hear well.</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>SYM_DIZZINESS</LocID>                                       <Text>Dizziness</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>SYM_DIZZINESS_DESCRIPTION</LocID>                           <Text>Temporary impairment in spatial perception and stability (vertigo, disequilibrium, presyncope, lightheadedness).</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>SYM_DIZZINESS_DESCRIPTION</LocID>                           <Text>The patient feels dizzy.</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>SYM_DRY_COUGH</LocID>                                       <Text>Dry cough</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>SYM_DRY_COUGH_DESCRIPTION</LocID>                           <Text>Sudden repetitive non-productive cough.</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>SYM_DRY_COUGH_DESCRIPTION</LocID>                           <Text>The patient coughs without producing any mucus or phlegm.</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>SYM_DYSPHAGIA</LocID>                                       <Text>Dysphagia</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>SYM_DYSPHAGIA_DESCRIPTION</LocID>                           <Text>Difficulty swallowing.</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>SYM_DYSPHAGIA_DESCRIPTION</LocID>                           <Text>The patient experiences difficulty or discomfort in swallowing.</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>SYM_DYSPNEA</LocID>                                         <Text>Dyspnea</Text>  </GameDBLocalizedString>
```

Comment:
There is no difference between "Dyspnea" and "difficulties breathing" that I changed in line